velox/dwio/parquet/tests/ParquetTableScanTest.cpp

```diff
@@ -36,53 +34,177 @@ class ParquetTableScanTest : public HiveConnectorTestBase {
     parquet::registerParquetReaderFactory();
   }
 
+  void assertSelect(
+      std::vector<std::string> outputColumnNames,
+      const std::string& sql) {
+    auto plan = PlanBuilder()
+                    .tableScan(
+                        ROW(std::move(outputColumnNames),
```

Comment:
this code is still duplicated in multiple places; is there any particular to reason for that? why not change the helper function to return the ROW type?

Reply:
@mbasmanova Just changed it to 
```
  std::shared_ptr<const RowType> getRowType(
      std::vector<std::string>&& outputColumnNames) const {
    std::vector<TypePtr> types;
    for (auto colName : outputColumnNames) {
      types.push_back(rowType_->findChild(colName));
    }

    return ROW(std::move(outputColumnNames), std::move(types));
  }
```

Reply:
Nice. You want to use RowTypePtr for the return type for brevity.